src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorDiagnosticConverter.cs

```diff
@@ -27,6 +28,7 @@ public static Diagnostic Convert(RazorDiagnostic razorDiagnostic, SourceText sou
         {
             Message = razorDiagnostic.GetMessage(CultureInfo.InvariantCulture),
             Code = razorDiagnostic.Id,
+            Source = "Razor",
```

Comment:
This makes the error list look a little nicer:
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/754264/225807494-2ad8355c-51d4-478c-ac26-07fe5121859f.png">